pkg/cmd/cli/cmd/deploy.go

```diff
@@ -227,7 +228,9 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	config.Status.LatestVersion++
 	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
 	if err == nil {
-		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "--> Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "    Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
+		fmt.Fprintf(out, "    Use '%s scale --replicas=N dc/%s' to scale it accordingly.\n", o.baseCommandName, config.Name)
```

Comment:
I don't know that this is the right place to suggest this.  Why would I
scale while a deployment is running?

On Tue, Apr 5, 2016 at 12:56 PM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/deploy.go
> <https://github.com/openshift/origin/pull/8360#discussion_r58574858>:
>
> > @@ -227,7 +228,9 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
> >  	config.Status.LatestVersion++
> >  	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
> >  	if err == nil {
> > -		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
> > +		fmt.Fprintf(out, "--> Started deployment #%d\n", config.Status.LatestVersion)
> > +		fmt.Fprintf(out, "    Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
> > +		fmt.Fprintf(out, "    Use '%s scale --replicas=N dc/%s' to scale it accordingly.\n", o.baseCommandName, config.Name)
>
> I am reluctant to suggest oc scale after starting a deployment but I
> wanted to discuss it. It would be a nice fit if the deployer scaled based
> on d.spec.replicas and not a fixed annotation. Currently, during a running
> deployment, the dc will be scaled up but the size will be persisted in the
> underlying deployment in the next sync interval. Proportional scaling would
> also be a nice-to-have. @ironcladlou <https://github.com/ironcladlou>
> thoughts?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8360/files/6fda297aedd34d67eb786c818ff0a59e7dcc0c7b#r58574858>
>